### ISSUE TITLE
Additional step of updating GuidSymbol is missing

### ISSUE THREAD

**Issue by Jebarson** [MicrosoftDocs/visualstudio-docs#2321]
There is an additional step after Step 2 is required. This is basically to change GuidSymbol name to the new group name that is created for the solution.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dbec6d43-3798-8794-59ff-7ffc8dc1a63d
* Version Independent ID: 8c8bf4ec-af85-dca1-a1f1-2e56d6345b5b
* Content: [Adding a Command to the Solution Explorer Toolbar - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/extensibility/adding-a-command-to-the-solution-explorer-toolbar?view=vs-2017)
* Content Source: [docs/extensibility/adding-a-command-to-the-solution-explorer-toolbar.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/extensibility/adding-a-command-to-the-solution-explorer-toolbar.md)
* Product: **visual-studio-dev15**
* GitHub Login: @gregvanl
* Microsoft Alias: **gregvanl**

**Comment by leslierichardson95** [comment]
Thanks for submitting your feedback and we apologize for the delayed response. Due to current resourcing and substantial extensibility changes we are currently working on that will result in significant doc changes, we are closing this lower priority issue. We will return to this issue on a later date and address it based on it's relevance after those extensive extensibility updates are made. Feel free to comment here as necessary.
